apps/src/applab/designElements/button.jsx

```diff
@@ -288,26 +291,22 @@ export default {
     element.appendChild(document.createTextNode('Button'));
     element.style.padding = '0px';
     element.style.margin = '0px';
-    if (experiments.isEnabled('applabThemes')) {
-      element.style.borderStyle = 'solid';
-      const currentTheme = elementLibrary.getCurrentTheme(
-        designMode.activeScreen()
-      );
-      const fontSize = this.themeValues.fontSize[currentTheme];
-      // Initial button size when fontSize is 14 or smaller is 80x30 (classic theme)
-      // Initial button size when fontSize is 15 or greater is 100x40 (new themes)
-      element.style.height = fontSize <= 14 ? '30px' : '40px';
-      element.style.width = fontSize <= 14 ? '80px' : '100px';
-      elementLibrary.applyCurrentTheme(element, designMode.activeScreen());
-    } else {
-      element.style.height = '30px';
-      element.style.width = '80px';
-      element.style.fontFamily = fontFamilyStyles[0];
-      element.style.fontSize = defaultFontSizeStyle;
-      elementUtils.setDefaultBorderStyles(element, {forceDefaults: true});
-      element.style.color = themeColor.buttonText.classic;
-      element.style.backgroundColor = themeColor.buttonBackground.classic;
-    }
+    element.style.borderStyle = 'solid';
+    const currentTheme = elementLibrary.getCurrentTheme(
+      designMode.activeScreen()
+    );
+    const fontSize = this.themeValues.fontSize[currentTheme];
+    const small = fontSize <= MAX_SMALL_FONT_SIZE;
```

Comment:
Should be isSmallFont or something similar. The current name 'small' sounds like this is an integer rather than boolean.

Reply:
Should be a "Nit: ..." I think? If we were to clean up variable names in the codebase, I'm not sure this is where I'd start!

That said, I can see the argument for doing it now, when transitioning from pilot -> shipped code.

Reply:
Yep, that's a NIT :)